cloudformation/frontend.yaml

```diff
@@ -1,22 +1,47 @@
 Resources:
   FrontEndApi:
-    Type: AWS::Serverless::HttpApi
-    Condition: FrontEndEnabled
+    Type: AWS::Serverless::Api
+    Condition: !And [FrontEndEnabled, !Not [IsEphemeral]]
     DependsOn:
       - FrontEndDomainName
       - FrontEndHostedZoneName
     Properties:
-      AccessLogSettings:
+      AccessLogSetting:
         DestinationArn: !GetAtt FrontEndApiLogGroup.Arn
         Format: '{ "requestId":"$context.requestId", "requestTime":"$context.requestTime", "routeKey":"$context.routeKey", "status":"$context.status" }'
       Auth:
         Authorizers:
           LambdaAuthorizer:
-            AuthorizerPayloadFormatVersion: '2.0'
-            EnableFunctionDefaultPermissions: true
-            EnableSimpleResponses: true
             FunctionArn: !GetAtt FrontEndAuthFunction.Arn
+            FunctionPayloadType: REQUEST
+            Identity: 
+              Headers: 
+                - Authorization
+              QueryStrings:
+                - code
+                - state
         DefaultAuthorizer: LambdaAuthorizer
+      CacheClusterEnabled: true
+      CacheClusterSize: '0.5'
+      DefinitionBody: 
+        openapi: 3.0.2
+        info:
+          title: DI Billing Frontend API
+          description: A frontend for the Program Delivery Team to manage billing across Digital
+            Identity
+          version: '1.0'
+        servers:
+        - url: https://btm.accounts.gov.uk
+        paths:
+          "/":
+            get:
+              description: The home page
+              responses:
+                '200':
+                  description: OK
+                  text/html:
+                    type: string
+                    example: "<h1>Welcome to DI Billing</h1>"
```

Comment:
I don't know to be honest, since the routing is handled by express rather than the gateway I think it should be ok. 

I'd like to have a more sustainable way to manage this definition though, it's good documentation if nothing else. There's a different way to do this where one provides `DefinitionUri` instead and this can point at a file in a bucket with this definition in it. It's also possible to generate these specs from modular declarations, that's how I created this one. 

I'll bring in another PR after this one that includes the modular spec, a script to build it, and a bucket to store it in.